tekton/manifest.yaml

```diff
@@ -1,8 +1,9 @@
 ---
 name: Tekton
-version: v0.24.1
+version: v0.44.0
```

Comment:
By curiosity where this version come from?

Reply:
From https://github.com/tektoncd/pipeline/blob/main/releases.md - and went with the latest "LTS" (Long Term Support) version.  Only indirectly implied in the Makefile version variable comment - I could add a bit more documentation around the decision if you wish.

Reply:
I do believe it's better to use fix version than relying on the latest version which may introduce unexpected behavior at the wrong time, that being said, the process to automate version 
update can be hard to automate. I am not affiliated to CIVO :), I was just curious to understand your update scenario.
In this specifc case, I was curious why using the latest version for tekton dashboard and tekton trigger, while using a specific version of tekton pipeline, and why the app.yaml seemed outdated but your answered my question :)

I am sorry if I confused you in any way